src/components/form/row.vue

```diff
@@ -21,7 +21,7 @@ except according to the terms contained in the LICENSE file.
     <td v-for="reviewState of visibleReviewStates" :key="reviewState" class="review-state">
       <template v-if="form.publishedAt != null">
         <link-if-can :to="formPath(form.projectId, form.xmlFormId, `submissions?reviewState=${urlFilterEncode.get(reviewState)}`)"
-          :title="$t(`reviewState.${reviewState}`)">
+          v-tooltip.no-aria="$t(`reviewState.${reviewState}`)">
```

Comment:
I think screen readers will read the column header when you enter a cell, so as long as the column headers are set up usefully, there doesn't need to be additional text describing the cell. Because of that, I chose `v-tooltip.no-aria` here. (For this table, we could probably update the column headers so that they're more useful to screen readers, but I think that's something we can do later.)